lms/static/js/spec/groups/views/cohorts_spec.js

```diff
@@ -255,6 +277,54 @@ define(['backbone', 'jquery', 'js/common_helpers/ajax_helpers', 'js/common_helpe
                 });
             });
 
+            describe("Course Cohort Settings", function () {
+                it('enable/disable working correctly', function () {
```

Comment:
These should read as a sentence starting with 'it', and most tests are about things working correctly. I would call it something like:

    it('can enable and disable cohorting', ...